newsboat.cpp

```diff
@@ -66,6 +67,24 @@ void print_usage(const std::string& argv0, const std::string& config_path,
 			_s("<configfile>"),
 			_s("read configuration from <configfile>")
 		},
+		{
+			'-',
+			"queue-file",
+			_s("<queuefile>"),
+			_s("use <queuefile> as podcast queue file")
+		},
+		{
+			'-',
+			"history-search",
+			_s("<searchfile>"),
+			_s("save the input history of the search functions to <searchfile>")
```

Comment:
There's only one search function in Newsboat, and we call it simply "search", so let's rephrase that a bit:

```suggestion
			_s("save the input history of the search to <searchfile>")
```